templates/collections/collection.html

```diff
@@ -0,0 +1,71 @@
+{% extends "simple_page.html" %}
+{% load static %}
+{% load bw_templatetags %}
+{% load display_sound %}
+{% load util %}
+
+{% block title%}Collections{%endblock title%}
+{% block page-title%}{{collection.name}}{%endblock page-title%}
+{% block page-content %}
+<div class="v-spacing-top-5">
+    <div class="row no-gutters">
+        <div class="col-lg-3 offset-lg-1 order-lg-last v-spacing-3">
+            <section class="bw-profile__section_stats v-spacing-top-4 async-section"
+                            data-async-section-content-url="{% url 'collection-stats-section' collection.id %}?ajax=1">
+                            <img width="12px" height="12px" src="{% static 'bw-frontend/public/bw_indicator.gif' %}">
+            </section>
+            <div class="v-spacing-top-4">
+                {% if is_owner or is_maintainer%}
+                <a class="no-hover btn-secondary display-inline-block w-100 text-center v-spacing-top-3" href="{% url 'edit-collection' collection.id %}" title="Edit collection">Edit collection</a>
+                {% endif %}
+                {% if collection.num_sounds > 0%}
+                <a class="no-hover btn-primary display-inline-block w-100 text-center v-spacing-top-3" href="{% url 'download-collection' collection.id %}" title="Download collection">Download Collection</a> 
+                {% endif %}
+            </div>
+        </div>    
+        <div class="col-lg-8">
+            {% if collection %}
+            <div class="middle bw-sound-page__user v-spacing-top-3">
+                <div class="h-spacing-1 ellipsis">
+                    {% bw_user_avatar collection.user.profile.locations.avatar.M.url collection.user.username 40 %}
+                </div>
+                <div class="h-spacing-left-1">
+                    <a href="{% url 'account' collection.user.username %}">{{ collection.user.username | truncate_string:15 }}</a>
+                    <p class="text-grey">{{ collection.created|date:"F jS, Y" }}</p>
```

Comment:
not part of this PR, but I see that this `date` format is used in lots of places in freesound. would be great to make our own custom filter that has the format built-in